src/eva/core/metrics/defaults/segmentation/multiclass.py

```diff
@@ -13,54 +13,40 @@ class MulticlassSegmentationMetrics(structs.MetricCollection):
     def __init__(
         self,
         num_classes: int,
-        average: Literal["macro", "weighted", "none"] = "macro",
-        ignore_index: int | None = None,
+        include_background: bool = False,
+        input_format: Literal["one-hot", "index"] = "index",
         prefix: str | None = None,
         postfix: str | None = None,
     ) -> None:
         """Initializes the multi-class semantic segmentation metrics.
 
         Args:
             num_classes: Integer specifying the number of classes.
-            average: Defines the reduction that is applied over labels.
-            ignore_index: Specifies a target value that is ignored and
-                does not contribute to the metric calculation.
+            include_background: Whether to include the background class in the metrics computation.
+            input_format: What kind of input the metrics should expect.
```

Comment:
Would be good to make the docstring for `input_format` a bit more specific.

Reply:
nice done